packages/studio-web/src/app/demo/demo.component.ts

```diff
@@ -160,6 +209,13 @@ export class DemoComponent implements OnDestroy, OnInit {
     if (this.selectedOutputFormat == "html") {
       await this.updateImages(ras);
       await this.updateTranslations(ras);
+      const timestamp = new Date()
+        .toISOString()
+        .replace(/[^0-9]/g, "")
+        .slice(0, -3);
+      const basename =
+        (this.slots.title ? slugify(this.slots.title, 15) : "readalong") +
```

Comment:
This is great, except it's too anglocentric: if the title is ᐃᓄᒃᑎᑐᑦ, the slug is empty and you get `-20240402212031.zip`, and the .txt file inside is similarly limited. Similarly "Éric" becomes "ric", but the Inuktitut example is a lot more dramatic. Can we slugify the text-unidecode output, maybe?

Reply:
thanks for this @joanise - I've updated it to use unidecode